app/Http/Controllers/Store/DashboardController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Controllers\Service;
+
+use App\Http\Controllers\Controller;
+use Auth;
+use URL;
+
+class DashboardController extends Controller
+{
+    /**
+     * Index the Dashboard options
+     */
+    public function index()
+    {
+        $user = Auth::user();
```

Comment:
These routes will be guarded at route level, right? So we don't need to check this user is the right type to have access here.

Reply:
Confirm that users who are not 'Store type/ Children centre users' will not be able to see this.